setup-cluster.sh

```diff
@@ -26,23 +39,23 @@ cat << EOF | tee ${VAGRANT_CONFIG}
 \$node_count = ${NODE_COUNT}
 \$master_count = 1
 
-\$vm_memory = 1024
-\$vm_vcpus = 1
-\$vm_disk = 45
+\$vm_memory = 16192
```

Comment:
16GB master with 8GB nodes is a production grade cluster, which is overkill for what we need. I've been using 1GB vms for 4 node multinode setup for a while. I haven't come close to using all the memory.

I found the minimum in the kubernetes docs a while back. This looks like an updated version of the doc:
```Apiserver and etcd together are fine on a machine with 1 core and 1GB RAM for clusters with 10s of nodes. Larger or more active clusters may benefit from more cores.```

https://kubernetes.io/docs/getting-started-guides/scratch/#designing-and-preparing